### ISSUE TITLE
Annotator has nothing left to annotate and marked as completed after only a small part of the annotations

### ISSUE THREAD

**Issue by johann-petrak** [GateNLP/gate-teamware#379]
I have set up an annotation project with two annotators and 8000 items to annotate. One of the annotators has started to work on this and after 520 annotations they get the message "nothing more to annotate". 

When I check the annotator list for the project, that user has the following row:
```
<name>  520/8000 Completed and the 4 buttons, but complete and reject are also grey.
```
Why is this happening? How can I make it so that the annotator gets ALL the remaining instances shown?  Why is it possible to mark an annotator as completed but not one that is completed as not completed so it can continue to annotated? Why 520? I am really puzzled about this. 

The About page shows this is Teamware 0.3.1 (this was the newest version when the project was set up, but since it got delayed it may now be an old version: is there information somewhere about how to upgrade if necessary?)

**Comment by johann-petrak** [comment]
Here are the project settings:

```
{
   "allow_annotation_change" : true,
   "allow_document_reject" : false,
   "annotation_timeout" : 60,
   "annotations_per_doc" : 1,
   "annotator_guideline" : "",
   "annotator_max_annotation" : 1,
   "can_annotate_after_passing_training_and_test" : true,
   "configuration" : [
      {
         "name" : "id",
         "text" : "<b>ID: {{ id }}</b>",
         "type" : "html"
      },
      {
         "name" : "text",
         "text" : "<div style='white-space: pre-wrap'>{{ text }}</div>",
         "type" : "html"
      },
      {
         "name" : "sensitive",
         "options" : [
            {
               "label" : "OK",
               "value" : "ok"
            },
            {
               "label" : "Ambiguous",
               "value" : "ambiguous"
            },
            {
               "label" : "NOK - Sensitive!",
               "value" : "nok"
            }
         ],
         "title" : "Sensitive",
         "type" : "radio"
      },
      {
         "name" : "comment",
         "optional" : true,
         "title" : "Comment",
         "type" : "textarea"
      }
   ],
   "description" : "",
   "document_gold_standard_field" : "gold",
   "document_id_field" : "id",
   "document_input_preview" : {
      "id" : "sdsdsdsdd",
      "text" : "<p>Some html text \n\n<strong>in bold</strong>.</p><p>Paragraph 2.</p>"
   },
   "document_input_preview_csv" : "",
   "document_pre_annotation_field" : "",
   "has_test_stage" : false,
   "has_training_stage" : false,
   "min_test_pass_threshold" : 1,
   "name" : "NAME"
}
```

**Comment by johann-petrak** [comment]
So the "maximum proportion of documents annotated per annotator" is set to 1 which I thought would mean that the annotator should annotate all documents. This was so far also the only annotator to annotate more than just a couple of documents, so most of the remaining (8000-520) documents have not been annotated by anyone. 
Is there any way how the annotator could have hit a wrong button and set herself to "completed"?

**Comment by johann-petrak** [comment]
Is this a bug or did I configure something wrong?

**Comment by johann-petrak** [comment]
Seems this may be caused by a bug that has been fixed already (perhaps #374 ? )

The workaround was to connect to the DB directly and update the required values to remove the completed status from the annotator.
